packages/components/mention/src/mention.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+  <div :class="ns.b()">
+    <el-input
+      v-bind="mergeProps(passInputProps, $attrs)"
+      ref="elInputRef"
+      :model-value="modelValue"
+      @input="handleInputChange"
+      @keydown="handleInputKeyDown"
+      @focus="handleInputFocus"
+      @blur="handleInputBlur"
+      @mousedown="handleInputMouseDown"
+    >
+      <template v-for="(_, name) in $slots" #[name]="slotProps">
+        <slot :name="name" v-bind="slotProps" />
+      </template>
+    </el-input>
+    <el-tooltip
+      ref="tooltipRef"
+      :visible="visible && (!!filteredOptions.length || loading)"
+      :popper-class="[ns.e('popper'), popperClass]"
+      :popper-options="popperOptions"
+      :placement="computedPlacement"
+      :fallback-placements="computedFallbackPlacements"
+      effect="light"
+      pure
+      :offset="offset"
+      :show-arrow="showArrow"
+    >
+      <template #default>
+        <div :style="cursorStyle" />
+      </template>
+      <template #content>
+        <el-mention-dropdown
+          ref="dropdownRef"
+          :options="filteredOptions"
+          :disabled="disabled"
+          :loading="loading"
+          @select="handleSelect"
```

Comment:
```suggestion
          @select="handleSelect"
          @click.stop="elInputRef?.focus"
```

Clicking the top and bottom of the dropdown will lose focus

Reply:
done